bookshelf/api_v3/gce.py

```diff
@@ -0,0 +1,425 @@
+from time import time, sleep
+import uuid
+
+from zope.interface import implementer, provider
+from pyrsistent import PClass, field
+from oauth2client.client import (
+    SignedJwtAssertionCredentials, GoogleCredentials
+)
+from googleapiclient import discovery
+from googleapiclient.errors import HttpError
+
+from bookshelf.api_v2.logging_helpers import log_green, log_yellow, log_red
+from bookshelf.api_v1 import wait_for_ssh
+from cloud_instance import ICloudInstance, ICloudInstanceFactory, Distribution
+
+
+class GCEConfiguration(PClass):
```

Comment:
Not really.
Only if you are really really bored.
If someone managed to get this far without reading the readme. Then they don't deserve any special assistance.